README.md

```diff
@@ -29,7 +29,7 @@ All developers are welcome to contribute to the codebase.  We ask that, if possi
 * Sendgrid (for outbound email).
 * SSL using [Let's Encrypt](letsencrypt.org) and [Substrakt](https://github.com/substrakt/letsencrypt-heroku) at [End Bias Certificate Manager](https://endbias-certificate-manager.herokuapp.com/).
 
-For setting up a local development environment [here](docs/README.md)
+For setting up a local development environment [here](https://github.com/EBWiki/EBWiki/blob/docs/docs/SETUP_LOCALLY.md)
```

Comment:
Does this not work with a relative link?  It should work if you link to `docs/SETUP_LOCALLY.md`

Reply:
Yeah, it will work with a relative link also

Reply:
Paarmita,  what's the status on this?  Is there anything blocking you from making this change?

Reply:
@rlgreen91 this link also work.So I think it's done.